packages/synapse-react-client/src/components/HasAccess/HasAccessV2.tsx

```diff
@@ -90,6 +100,13 @@ function AccessIcon(props: { downloadType: FileHandleDownloadTypeEnum }) {
   return <></>
 }
 
+function isFileHandleDownloadTypeUndefined(
+  fileHandleDownloadType: FileHandleDownloadTypeEnum | undefined,
+) {
+  // note, this can't be "if (!downloadType)" since DownloadTypeEnum has a 0 value (which is falsy)
+  return fileHandleDownloadType === undefined
```

Comment:
Is this really true? `FileHandleDownloadTypeEnum` is a string enum

Reply:
I see now that this comment was just copied.

Reply:
Here's the original change where the comment should have been removed: https://github.com/Sage-Bionetworks/synapse-web-monorepo/commit/fd9c270d53f6c3f2042809ba8b14e24679ee9c90

Can you go ahead and remove this comment?

Reply:
Oh will do! Thanks for tracking down the original change.